net/tstun/wrap.go

```diff
@@ -920,6 +923,9 @@ func (t *Wrapper) Read(buffs [][]byte, sizes []int, offset int) (int, error) {
 		if !t.disableFilter {
 			response := t.filterPacketOutboundToWireGuard(p, pc)
 			if response != filter.Accept {
+				metricOutboundDroppedPacketsTotal.Add(dropPacketLabel{
+					Reason: DropReasonError,
```

Comment:
I think we'd want to increment the error counter earlier in this function, when we return `res.err`